src/Sylius/Behat/Page/Admin/Crud/UpdatePage.php

```diff
@@ -49,6 +49,19 @@ public function saveChanges()
     /**
      * {@inheritdoc}
      */
+    public function checkValidationMessageFor($element, $message)
+    {
+        $foundedElement = $this->getElement($element)->getParent()->find('css', '.pointing');
+        if(null === $foundedElement) {
+            throw new ElementNotFoundException(sprintf('Element %s was not found, so asserting its validation message was not possible.', $element));
```

Comment:
Wdyt about ``Element %s has not been found, so asserting its validation message is not possible.``?